### ISSUE TITLE
Is it possible to open just a Terminal tab instead of a new window?

### ISSUE THREAD

**Issue by lifeBalance** [karan/atom-terminal#60]
In Sublime text I use a plugin, I think it's called MacTerminal, that allows me to open files in the Terminal, but if the Terminal is already open, it just adds a **tab**. 
Is that possible to achieve in atom-terminal with some setting?
I really find annoying a **new window** terminal every time.

**Comment by jacwah** [comment]
Such a feature would have to be custom made for each terminal application available, like MacTerminal.

The terminals I open through atom-terminal are very short-lived. I run one or a few commands, and as soon as I want to focus Atom, I press Ctrl-D to close the terminal. This way I don't have to worry about a growing number of them, and can drop in and out of the shell just using the keyboard.

**Comment by jirsbek** [comment]
:+1: +1 I would love to open Terminal in new tab.

**Comment by tkw1536** [comment]
I agree with @jacwah, we would have to write code seperatly for each terminal out there. 
If you want to open a new tab instead of a new terminal, you can always write yourself a custom script to do it and then set that under the "app" setting.

**Comment by jirsbek** [comment]
Oh right, I understand and agree. I forgot there is not only one Terminal app and one OS. Custom package is a way to go.

**Comment by vorg** [comment]
@lifeBalance @jirsbek here you go https://atom.io/packages/atom-mac-terminal

**Comment by lifeBalance** [comment]
Very good Marcin! Thanks a lot. Now, it would be awesome if you finish it a bit; I mean it does not open a new tab, just the file or directory in whatever tab you have open in Terminal.

**Comment by vorg** [comment]
@lifeBalance Heh, the whole point was to open a new tab. I guess we could add an option to skip adding new tab in the settings. Please add the idea here and let's discuss https://github.com/vorg/atom-mac-terminal/issues/
